ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-joins.json

```diff
@@ -1227,41 +1238,41 @@
       ],
       "post": {
         "topics": {
-          "topics" : [
+          "topics": [
             {
-              "name" : "_confluent-ksql-some.ksql.service.idquery_CSAS_OUTPUT_0-L_Join-left-repartition",
-              "keyFormat" : {"format" : "JSON_SR", "features": ["UNWRAP_SINGLES"]},
-              "valueFormat" : {"format" : "JSON"}
+              "name": "_confluent-ksql-some.ksql.service.idquery_CSAS_OUTPUT_0-KafkaTopic_Right-Reduce-changelog",
```

Comment:
This topic should also go away with https://github.com/confluentinc/ksql/issues/6650 -- it makes sense that all internal topics should be Avro formatted in this case.